scripts/convert-ESO-PV-forecasts-from-CSV-to-HDF5.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+This script converts the multiple CSV files from ESO into a single NetCDF file.
+
+The output NetCDF file has two DataArrays: 'ASL' and 'ML'.
+
+ESO uses two PV forecasting algorithms in the Platform for Energy Forecasting (PEF):
+
+* ASL: "Advanced Statistical Learning": These are hand-crafted models written in `R` by the
+       ESO forecasting team.
+* ML: "Maching Learning": ML models, trained from historical data, created by the ESO Labs team.
+
+Each DataArray has three dimensions:
+
+1. `gsp_id`: A string identifying the Grid Supply Point region.
+2. `forecast_date_time`: The UTC datetime when ESO ran their forecast.  Not exactly the
+    same as the NWP init time.  This script takes the floor('30T') of the
```

Comment:
NWP?

Reply:
Sorry, it is a bit confusing!  I've added more context and detail to the docstring!